utils/mkrest.py

```diff
@@ -81,8 +81,9 @@ def read_file(name):
 if tmp_data:
     sys.stdout.write(tmp_data)
 
+arguments = ['pandoc', '-s', '-r', 'html', src_file, '-w', 'rst']
 process = subprocess.Popen(
-    "pandoc -s -r html %s -w rst" % src_file, shell=True, stdout=subprocess.PIPE
+    arguments, stdout=subprocess.PIPE
```

Comment:
As I reviewed in https://github.com/OSGeo/grass/pull/3436#discussion_r1493954603, it was all good for me!